### DISCUSSION TITLE
Encrypted Storage Library

### DISCUSSION THREAD

**Discussion by aymane-missouri-nw** [infinitered/ignite#2380] [Q&A] [answered]
Why did you remove react-native-keychain from the boilerplate , isn't necessary to have an encrypted storage library in the boilerplate ?

**Comment by frankcalise** [comment, marked as answer]
Hey @aymane-missouri-nw - thanks for the question.

`react-native-keychain` got removed because it wasn't always our go to for our client projects.

You can always add it back in if it suits your needs.

Also, if you want to go with MMKV as an option, we have a recipe for that over on the [Ignite Cookbook](https://ignitecookbook.com/docs/recipes/MigratingToMMKV)
